src/components/Dissolution/AssociationDetails.vue

```diff
@@ -134,6 +162,13 @@ export default class AssociationDetails extends Mixins(CommonMixin, EnumMixin) {
     return `${this.getCorpTypeDescription(this.getEntityType)}`
   }
 
+  /** The business start date. */
+  get businessStartDate (): string {
+    // SB TODO: get data from object and pass here
```

Comment:
When can you do this?

Reply:
Once we have API available, will update the component, we have ticket on upcoming sprint